manifests/init.pp

```diff
@@ -89,8 +89,10 @@
   Pattern[/^(?:-)?[0-9]$/] $debug_level = '-2',
   Pattern[/^[0-9]+$/] $randomwait = '360',
   Array $exclude_packages = [],
+  String $mailfrom = 'root@example.com',
```

Comment:
```suggestion
  String $email_from = 'root@example.com',
```

Reply:
Prefer DNF naming as yum_cron will go away soonish as RHEL7 is EOL for those not paying for ELS.